### DISCUSSION TITLE
Exporting documentation

### DISCUSSION THREAD

**Discussion by Sid8899** [spirit-code/spirit#612] [Q&A] [answered]
Hi, 

Equations in the documentation does not appear : in .mk files after build on windows, and on the readthedocs website https://spirit-docs.readthedocs.io/en/latest/core/docs/Definitions.html . But it works from GitHub repo website, there is a way to download a pdf of the docs directly from GH and still have equations ?

**Comment by GPMueller** [comment, marked as answer]
This does not seem like a "discussion" but an issue, so I've moved it to #613.
